databases/week2/DB_W2_HW.sql

```diff
@@ -0,0 +1,37 @@
+-- 1. Add a task with the these attributes: title, description, created, updated, dueDate, statusID, userID
+INSERT INTO task 
+(title, description, created, updated, due_Date, status_id, user_id)
+VALUES 
+('Make a new task' ,
+'Make a new task in task table',
+NOW(),
+NOW(),
+NOW() + INTERVAL 5 DAY,
+1,
+1);
```

Comment:
Really cool that you are using INTERVAL combined with NOW()! :) 

My only suggestion would be to indent the column names and the values so they stand out visually. Something like this:

```sql
INSERT INTO task (
    title, description, created, updated, due_date, status_id, user_id
) VALUES (
    'Make a new task' ,
    'Make a new task in task table',
    NOW(),
    NOW(),
    NOW() + INTERVAL 5 DAY,
    1,
    1
);
```

Reply:
Hi Flemming,
Thanks a million for your advises. I will keep that in my mind.